packages/dd-trace/src/exporters/common/request.js

```diff
@@ -74,7 +83,11 @@ function request (data, options, keepAlive, callback) {
       onError(err)
     })
 
-    dataArray.forEach(buffer => req.write(buffer))
+    if (isFormData) {
+      data.pipe(req)
+    } else {
+      dataArray.forEach(buffer => req.write(buffer))
+    }
```

Comment:
the changes here is so that we can use `request` with an instance of `FormData`

Reply:
Let's do something like `isStream` of `isReadable` instead since it doesn't necessarily need to be form data.